core/src/test/java/tachyon/hadoop/contract/TachyonFSContract.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package tachyon.hadoop.contract;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.contract.AbstractFSContract;
+import org.apache.hadoop.fs.contract.ContractOptions;
+import org.apache.hadoop.fs.contract.ContractTestUtils;
+import org.apache.hadoop.util.Shell;
+import tachyon.Constants;
+import tachyon.hadoop.TFS;
+import tachyon.master.LocalTachyonCluster;
+
+import java.io.IOException;
+import java.net.URI;
+
+public class TachyonFSContract extends AbstractFSContract {
```

Comment:
Do you mean TFS and TFSFT? If so, the implementation being used is TFS but the real logic is in AbstractTFS which they both use, so I think its okay to just use TFS here.